api-reference/v2/live/callback/audio-chunk-ack.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: Audio chunk ack
```

Comment:
```suggestion
title: Audio chunk acknowledgement
```
We should refrain from using abbreviations or acronyms inside of the documentation